package/firmware/layerscape/ls-ddr-phy/Makefile

```diff
@@ -0,0 +1,49 @@
+#
```

Comment:
SDPX Header?

Reply:
Let me add it.

Reply:
Please move it to first line:

https://spdx.github.io/spdx-spec/appendix-V-using-SPDX-short-identifiers-in-source-files/#format-for-spdx-license-identifier

Reply:
Ok.